packages/-serializer-encapsulation-test-app/tests/integration/normalize-test.js

```diff
@@ -83,7 +83,7 @@ module('integration/serializer - normalize method forwards to Serializer#normali
 
     const store = this.owner.lookup('service:store');
 
-    assert.expectAssertion(() => {
+    assert.throws(() => {
```

Comment:
Why does this one change? Is it always an error (e.g. not an assertion)?

Reply:
its not run in production and it doesn't share the infra that ember-data's tests have